src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Models/ResetPasswordModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace TodoTemplate.App.Models
+{
+    public class ResetPasswordModel
+    {
+        [Required(ErrorMessage = "Please enter your new password.")]
+        [MinLength(6, ErrorMessage = "The password must have at least 6 characters.")]
+        public string? NewPassword { get; set; }
+
+        [Required(ErrorMessage = "Please enter your confirmation of the new password.")]
+        [Compare(nameof(NewPassword), ErrorMessage = "Password confirmation doesn't match Password.")]
```

Comment:
The `ErrorMessage` should be "Password confirmation doesn't match the new password." :)